runtime/src/main/java/org/corfudb/runtime/view/Layout.java

```diff
@@ -329,8 +329,8 @@ public void validateSegmentSeal(LayoutSegment layoutSegment,
             }
 
             @Override
-            public int getMinReplicationFactor(Layout layout) {
-                return 2;
+            public int getMinReplicationFactor(Layout layout, LayoutStripe stripe) {
```

Comment:
This might be a bit unrelated, but why isn't the LayoutStripe part of the Layout definition?

Reply:
Layout stripe is a part of the layout. We can just pass the stripe index and infer the stripe from the passed layout object. I felt this would be more intuitive since we already have the stripe.

Reply:
Ok, I see I was a bit confused because both were passed separately. But then I agree passing the stripe is more intuitive than the stripe index.